share/wake/lib/system/runner.wake

```diff
@@ -263,18 +263,14 @@ export def makeJSONRunner ((JSONRunnerPlan rawScript extraArgs extraEnv estimate
                 Nil
             )
 
-        require Pass build =
-            mkdir ".build"
```

Comment:
Where does this mkdir live now?

Reply:
Inside the `write`:

```
export def write (path: String) (content: String): Result Path Error =
    def spath = simplify path

    require Pass parent =
        simplify "{spath}/.."
        | mkdir # Note that this is recursive, as well.

    writeImp (parent, Nil) 0644 spath content
```

My guess is that the `mkdir` was originally split out purely for the error message, but the shell's `mkdir` already advertises the path it tried unsuccessfully to create, so keeping it separate didn't seem to have any benefit.

Also, CI apparently succeeded?  I'll look into the supposedly end-of-lifed image in a bit.